src/aggregator/server/http/server.go

```diff
@@ -73,7 +72,6 @@ func (s *server) ListenAndServe() error {
 func (s *server) Serve(l net.Listener) error {
 	mux := http.NewServeMux()
```

Comment:
you’ll need to make this http.DefaultServeMux or else the pprof handlers won’t be registered (since the default pprof _ import registers the handlers on the default server mux).